docs/reference/ml/anomaly-detection/ml-delayed-data-detection.asciidoc

```diff
@@ -16,6 +16,11 @@ if it is set too high, analysis drifts farther away from real-time. The balance
 that is struck depends upon each use case and the environmental factors of the
 cluster.
 
+IMPORTANT: If you get an error that says 
```

Comment:
IMPORTANT: If you get an error that says 
Datafeed missed XXXX documents due to ingest latency', consider increasing the value of `query_delay'. If this doesn't seem to help, investigate the amount of ingest latency and its cause. You can do this by comparing event and ingest timestamps.  High latency is often caused by bursts of ingested documents, misconfiguration of the ingest pipeline, or misalignment of system clocks.